src/components/ResumeList/ResumeList.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from 'react';
+import { ResumeListProps } from './type';
+import ResumeListDefault from './ResumeListDefault';
+import ResumeListMobile from './ResumeListMobile';
+
+type MobileProps = 'true' | 'false';
+type ResumeListPropsMobile = ResumeListProps & { isMobile: MobileProps };
```

Comment:
You can use `boolean` as `isMobile` type and then pass `true` or `false` without apostrophes